### PR TITLE
Sdlc update

### PR DESCRIPTION
**Pull request by morrisco**
### Give us some context

* What problems does this PR solve?
  * This is an update to the existing content and wording of the SDLC with input from legal team
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### Are you making a change to site code?

If you're changing site code (rather than the content of a doc), please follow
[conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

### CODE REVIEW

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx [hunk 1]
```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
 </table>
 
 ## Requirements
-Each project team is assigned a security engineer charged with ensuring the security of New Relic products. In the requirements building phase, the security engineer performs a risk assessment and then adds security requirements for the project. We add privacy and compliance experts to the project teams as needed.
+Each project team for our generally available New Relic One platform is assigned a security engineer charged with reviewing and advision on the security of New Relic products. In the requirements building phase the security engineer performs a risk assessment and then adds security requirements for the project. Privacy and compliance experts are added to the project teams as needed.
```
Comment (rhetoric101) [inline src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx:79]: @morrisco, could you clarify "reviewing and advision on"? Is this a typo?

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx [hunk 2]
```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
...
 
 ## Design
-During the design phase, the New Relic Security team collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds detailed specifications for the required controls to the project.
+During the design phase, the New Relic security engineer collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds  detailed specifications for the required controls to the project.
```
Comment (rhetoric101) [inline src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx:82]: @morrisco, looks like an extra space following "adds."

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx [hunk 3]
```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
...
+Each product engineer receives secure coding training which includes topics such as the [OWASP top 10](https://owasp.org/www-project-top-ten/), input sanitization, and using the secure frameworks and process already in place at New Relic. In the build phase the engineering team implements appropriate security features in the project following secure coding standards at New Relic.
 
 ## Verification
-Once feature complete, every pull request must be code reviewed by another engineer with write access to the repository. At the project level, teams may request a final review of the project before deploying. The security engineer verifies that the safeguards and controls discovered and documented in the risk assessment, requirements, and design phases have been addressed.
+Once code complete, every pull request must be code reviewed by another engineer with write access to the code repository. At the project level teams may request a final review of the project before deploying. The security engineer verifies the safeguards and controls discovered and documented in the risk assessment, requirements, and design phase have been addressed.
```
Comment (rhetoric101) [inline src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx:88]: I would rearrange that last sentence like this: The security engineer verifies the following have been addressed: safeguards and controls discovered and documented in the risk assessment, requirements, and design phases.

If this isn't accurate, let me know.

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx [hunk 4]
```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
...
 
-All New Relic code has passed static code and composition analysis to look for vulnerabilities in the code and dependencies. All files published by New Relic will include their calculated hash and a digital signature. New Relic is in the process of including hashes and signatures, so that anyone downloading files published by New Relic will be able to confirm their downloaded file has not been tampered with and is identical to the one published by New Relic.
+All New Relic code has passed static code and composition analysis to look for vulnerabilities in the code and dependencies. All files published by New Relic will include their calculated hash and a digital signature. New Relic is in the process of including hashes and signatures, anyone downloading files published by New Relic will be able to confirm their downloaded file has not been tampered with and is identical to the one published by New Relic.
```
Comment (rhetoric101) [inline src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx:90]: I'd separate these two ideas into separate sentences. How about this:

New Relic is in the process of including hashes and signatures.  Anyone downloading files published by New Relic will be able to confirm their downloaded file has not been tampered with and is identical to the one published by New Relic.

Just to clarify, "Is in the process" means New Relic doesn't do it yet, right? Also, this same comment about published files appears down in Deploy. Is this redundancy necessary?

**Review by rhetoric101 — COMMENTED** [review]
@morrisco, these changes looked good! I just had a few small suggestions and questions.

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx (hunk 2 above)
Comment (morrisco) [reply src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx]: Fixed in recent commit

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx (hunk 3 above)
Comment (morrisco) [reply src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx]: Updated wording here with additional input from legal.

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx (hunk 4 above)
Comment (morrisco) [reply src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx]: Done

File: src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx (hunk 1 above)
Comment (morrisco) [reply src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx:79]: This came from the collaboration with legal, I can sync up with them again. Is the issue clarity?

Comment (morrisco) [reply src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx]: That was a typo. Fix commited in the PR

**Review by rhetoric101 — APPROVED** [review]
@morrisco, these changes look good! Sorry the the delay!

**Pull request merged**